lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1437,6 +1437,23 @@ OpFoldResult AtenBoolIntOp::fold(FoldAdaptor adaptor) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenAnyBoolOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenAnyBoolOp::fold(FoldAdaptor adaptor) {
+  SmallVector<bool> inputList;
+  if (!matchPattern(getSelf(), m_TorchListOfConstantBools(inputList)))
+    return nullptr;
```

Comment:
I think we can implement a more aggressive folder in future. We can match a `torch.List.Construct` and if we find that any operand of `torch.List.Construct` is `torch.constant.bool true`, just fold it to `true`.

Reply:
Fixed